include/signal.h

```diff
@@ -267,9 +267,9 @@
  * special meaning in some circumstances (e.g., kill()).
  */
 
-#ifndef __SIGSET_T_DEFINED
```

Comment:
After some thought, your modify this just because you include ucontext.h directly? if so, it's better to don't bring ucontext.h into NuttX space. then we don't need modify this file anymore.

Reply:
Right, this is why I needed this change

Reply:
That's why it's better to not mix the NuttX and host header file which introduce many headache. And only allow source code in HOSTSRC can include the host header file.